assets/js/translator.js

```diff
@@ -1281,5 +1323,6 @@ function setRecentDstLangs(langs /*: string[] */) {
 /*:: import {ENTER_KEY_CODE, HTTP_BAD_REQUEST_CODE, HTTP_OK_CODE, SPACE_KEY_CODE, XHR_DONE, XHR_LOADING} from "./util.js" */
 /*:: import {persistChoices, restoreChoices} from "./persistence.js" */
 /*:: import {localizeInterface, getLangByCode, getDynamicLocalization, locale, iso639Codes, langDirection} from "./localization.js" */
+/*:: import {languages, iso639CodesInverse} from "./localization.js" */
```

Comment:
Please combine this with the above. Also, the problem might be that that `localization.js` doesn't export them. Have you verified they do?

Reply:
When i combined them eslint check said to too many in one line error.

Reply:
Oops. Oh I have to export them. :)

Reply:
What to do combine or not?

Reply:
You can split it into multiple lines without having multiple `import` statements. There's an example of that on L1317.

Reply:
i correctly exported and imported i think, but its still returns error

Reply:
Ah, right, I think it also needs to be added to the `global` declaration at the top to satisfy ESLint. The exports/imports are for Flow.

Reply:
Like i did before?

Reply:
Not exactly. I'm referring to https://github.com/apertium/apertium-html-tools/pull/352/commits/27ecdc01b1c5835b2d4ca130afee5b646e288cfe#diff-8331c23a07e5eb815c5b4393e81715efL27.

Reply:
Okay got it. :)